acnportal/acnsim/simulator.py

```diff
@@ -204,6 +206,8 @@ def _increase_width(a, target_width):
     Returns:
         numpy.Array
     """
+    if target_width <= len(a[0]):
+        return a
     new_matrix = np.zeros((len(a), target_width))
```

Comment:
Shouldn't this be a.shape[0]? len(a) would be the total number of elements in a, not the number of rows.

Reply:
For numpy arrays `len(a)` gives the number of rows. But I agree, in general using `a.shape` is clearer as it is more explicit about which dimension we're looking at.  I made the following changes:
`len(a)` -> `a.shape[0]`
`len(a[0])` -> `a.shape[1]`
There was also an instance in `ChargingNetwork` where `len` was used for a numpy array instead of `shape`.